packages/amplify-graphql-api-construct-tests/src/__tests__/backends/amplify-table/simple-todo/updateIndex/app.ts

```diff
@@ -49,7 +49,7 @@ todoTable.setGlobalSecondaryIndexProvisionedThroughput('byName2', {
   writeCapacityUnits: 4,
 });
 todoTable.pointInTimeRecoveryEnabled = true;
-todoTable.sseSpecification = { sseEnabled: false };
+todoTable.sseSpecification = { sseEnabled: true };
```

Comment:
Please add a comment--at least a doc callout--for future developers who might get confused by the API here.

Reply:
Added the comment to `amplify-table-1.test.ts` file.